packages/map-popup/i18n/fr.yml

```diff
@@ -5,5 +5,6 @@ otpUi:
     floatingBike: "Vélo flottant : {name}"
     floatingCar: "{company} {name}"
     floatingEScooter: Trottinette {name}
+    popupTitle: "{stationNetwork, select, false {{name}} other {{stationNetwork} {name}}}"
```

Comment:
If `popupTitle` is going to be telegraphic style at first, does it need to be in the language files?

Reply:
Same as above. I've made it backwards compatible so languages can adjust when they're ready. Should we consider removing the strings from other languages?